app/assets/stylesheets/application.scss

```diff
@@ -26,9 +26,39 @@ $colors: (
 $theme-colors: $colors;
 
 @import "bootstrap/scss/bootstrap";
-@import url('https://fonts.googleapis.com/css?family=Crimson Text');
-@import url('https://fonts.googleapis.com/css?family=Pathway Gothic One');
-@import url('https://fonts.googleapis.com/css?family=Karla');
+@import url('https://fonts.googleapis.com/css?family=Crimson+Text:400&subset=latin');
+@import url('https://fonts.googleapis.com/css?family=Pathway+Gothic+One&subset=latin');
+@import url('https://fonts.googleapis.com/css?family=Karla&subset=latin');
+
+html {
+  height: 100%;
+}
+
+body {
```

Comment:
We have two different body sections in this file. While I know they are additive, I'd like to merge them into one section.